.github/workflows/checks.yaml

```diff
@@ -72,6 +72,8 @@ jobs:
       run: make cluster-sync
 
     - name: Run e2e tests
+      env:
+        E2E_TEST_ARGS: --procs=1
```

Comment:
you can just remove the -p flag [here](https://github.com/kubevirt/ipam-extensions/blob/main/Makefile#L86)

Also, can you explain more why they need to run serially? which resources are constrained, and where?

Reply:
Removed this commit. Resolving this comment.